cves/kernel/CVE-2020-12768.yml

```diff
@@ -55,7 +55,9 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: | 
+  A one-time memory leak at the boot was discovered. The size is negligible,
+  and it can't be triggered at will. They forgot to free the page held.
```

Comment:
Consider being a little more descriptive of the problem here especially for those who don't know much about C